HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -189,4 +205,9 @@ def customizeHLTforCMSSW(process, menuType="GRun"):
     process = customiseFor18559(process)
     process = customiseFor19098(process)
 
+    if (menuType == "GRun2016"):
+        process = customiseFor19181_pixel_phase0(process)
+    else:
+        process = customiseFor19181_pixel_phase1(process)
```

Comment:
I only found phase0 configurations in GRun2016 configs. An HLT expert should comment